uninstall-hook.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# uninstall-hook.sh
```

Comment:
I would call this `uninstall-all-hooks.sh` since it's going to remove ALL instances of space commander (can have multiple installs in the same share precommit hook file)